react-app/src/navigation/AppRouter.tsx

```diff
@@ -23,23 +23,23 @@ const AppRouter: React.FC = () => {
   const wallet = useWallet();
   return (
     <BrowserRouter>
-        <Routes>
-          <Route element={<AppSideBarOutlet />}>
-            <Route path={AppRoutes.Overview} element={<OverviewScreen />} />
-            <Route path={AppRoutes.Dummy} element={<DummyScreen />} />
-            <Route path={AppRoutes.Proposals} element={<ProposalScreen />} />
-            <Route
-              path={AppRoutes.NewProposal}
-              element={<CreateProposalScreen />}
-            />
-            <Route
-              path={AppRoutes.ProposalDetail}
-              element={<ProposalDetailScreen />}
-            />
-          </Route>
+      <Routes>
+        <Route element={<AppSideBarOutlet />}>
+          <Route path={AppRoutes.Overview} element={<OverviewScreen />} />
+          <Route path={AppRoutes.Dummy} element={<DummyScreen />} />
+          <Route path={AppRoutes.Proposals} element={<ProposalScreen />} />
+          <Route
+            path={AppRoutes.NewProposal}
+            element={<CreateProposalScreen />}
+          />
+          <Route
+            path={AppRoutes.ProposalDetail}
+            element={<ProposalDetailScreen />}
+          />
+        </Route>
```

Comment:
Do you have `Format on Save` enabled on your vscode? And I prefer fix up if there's no code change

Reply:
okay let me enable it